### ISSUE TITLE
1.19.3 is not supported

### ISSUE THREAD

**Issue by TheCodeAddiction** [barneygale/quarry#191]
Quarry does not work with version `1.19.3`. It only works with `1.19.2` atleast when using quarry as a proxy.

In my example, I use the _hide_chat_proxy.py_. if the server uses `1.19.2`, it works great! If it uses the latest minecraft version, `1.19.3`, the proxy prints the following error:
![Screenshot_3](https://user-images.githubusercontent.com/55413550/212832576-3e99b0fb-fc77-46d9-ab2b-32810eab3688.png)
`No name known for packet: (761,'status','upstream',0)`

**Comment by davidawesome02** [comment]
yes, this project is basicly dead, the owner says he will not work on it much any more as it was a side project, and he has like to make money. If you wana fix it you can but he has not changed anything in 4 months, and no significant changes in a year or more

**Comment by MajliTech** [comment]
So no 1.19.4? Well, is there any project like this?

**Comment by ajh123** [comment]
> So no 1.19.4? Well, is there any project like this?

There is https://github.com/py-mine/mcproto, however it does not offer the a high level API like quarry does, but that seems the be the goal in the future.
